app/src/main/java/hm/orz/chaos114/android/tumekyouen/KyouenActivity.java

```diff
@@ -33,32 +33,49 @@
 import hm.orz.chaos114.android.tumekyouen.util.ServerUtil;
 import hm.orz.chaos114.android.tumekyouen.util.SoundManager;
 
-@NoTitle
-@EActivity(R.layout.main)
 public class KyouenActivity extends FragmentActivity {
+
+    private static String EXTRA_TUME_KYOUEN_MODEL
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName